test/unit/org/apache/cassandra/index/sai/utils/LongArrays.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.cassandra.index.sai.utils;
+
+import java.util.Arrays;
+
+import org.apache.cassandra.index.sai.disk.v1.LongArray;
+
+public final class LongArrays
```

Comment:
`LongArray` belongs to the package `org.apache.cassandra.index.sai.disk.v1`. The only user of this utility class, `OffsetFactoryTest`, also belongs to that package. So probably this class should also be there.